app/javascript/mastodon/locales/ru.json

```diff
@@ -194,7 +194,7 @@
   "notifications.column_settings.reblog": "Продвижения:",
   "notifications.column_settings.show": "Показывать в колонке",
   "notifications.column_settings.sound": "Проигрывать звук",
-  "notifications.group": "{count} notifications",
+  "notifications.group": "{count} уведомл.",
```

Comment:
is it possible to use correct plurals here? Something like "trends.count_by_accounts" key (`"trends.count_by_accounts": "{count} {rawCount, plural, one {person} other {people}} talking"`)